### DISCUSSION TITLE
Should we generate some solutions with Chat GPT?

### DISCUSSION THREAD

**Discussion by marghidanu** [PlummersSoftwareLLC/Primes#892] [Polls]
It be interesting if we could squeeze some optimizations on the existing solutions using Chat GPT, what do you guys think of this?

**Comment by rbergen** [comment]
I don't have a strong opinion about this in itself, but I do strongly feel that we should treat ChatGPT-generated solutions/optimizations the same way we have done those contributed/proposed by (other) contributors. 

Over the past couple of years, we've settled on a few ground rules when it comes to adding new solutions, and accepting changes to existing ones:

- A new solution either has to add a new language, or use a significantly different new approach to solutions already in the codebase. In fact, they should preferably be faster than the existing solution for the language in question.
- When it comes to proposed changes to existing solutions, the original authors of solutions have a say in changes to their contributions, if they choose to respond to suggested changes. 
- If original authors don't respond, the main criterion for accepting changes is that they objectively increase the speed (number of passes) of the solution a change is proposed to, and the main characteristics of the solution don't change. 

I think all of those rules should still apply to any additions/changes made to this project, whether aided by ChatGPT (or any other AI model) or not.
